en/contributing/test.md

```diff
@@ -68,7 +68,7 @@ make run_integration_tests
 Make sure you are connected to a vehicle and check the connection using e.g.:
 
 ```
-make && build/default/dronecore-integrationtests --gtest_filter="SitlTest.TelemetryAsync"
+make && build/default/dronecore-integrationtests --gtest_filter="SitlTest.TelemetryAsync"  
```

Comment:
@julianoes This doesn't work. 
1. How do we run this? I tried `make && build/default/integration_tests --gtest_filter="SitlTest.TelemetryAsync"` but no luck
2. How do we do this for external integration tests?

Reply:
1. `make && build/default/integration_tests/integration_tests_runner --gtest_filter="SitlTest.TelemetryAsync"`
2. WIP
3. See 1.